### DISCUSSION TITLE
Additional silent batch job types

### DISCUSSION THREAD

**Discussion by Nouneds** [Micke-K/IntuneManagement#163] [Ideas]
Hello,
as we are started to use export and import options for multiple environments I had a thought about covering other type of operations through pipelines. Do You have any plans to add more silent job types using app registrations ? It would greatly improve possibilities to work with different tenant. Even simple delete function would be great as import and export could cover for other stuff. 

Compare feature that actually looks into all environments using app registrations would also be great :)

**Comment by Micke-K** [comment]
Hello!

I'll see what I can do. I've been thinking about this for a while but not implemented it since it will require redesign of the authentication part and token management. 

Cheers!
